src/psqlgraph/query.py

```diff
@@ -212,8 +212,122 @@ def path(self, *paths):
         assert (
             not self.entity().is_abstract_base()
         ), "Please narrow your search by specifying a node subclass"
+        # TODO: This is the original
         for e in entities:
             self = self.join(*getattr(self.entity(), e).attr)
```

Comment:
This is not obvious but its creating two join statements, one for the edge and the other for the destination node. Not clear if both joins will need an alias, but pretty sure the destination node needs one.
I played with this a little bit and was able to get the existing tests to pass
```python
 relation = getattr(self.entity(), e)
 alias = aliased(Node.get_subclass_named(relation.target_class.__dst_class__))
 this = self.join(relation.attr[0], relation.attr[1].of_type(alias))
```
I also had to make an update to `self.entity()` to cater for when the join_point is an AliasedClass not an entity.